homeassistant/components/ld2410_ble/binary_sensor.py

```diff
@@ -0,0 +1,116 @@
+"""LD2410 BLE integration light platform."""
+
+from typing import Any
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from . import LD2410BLE
+from .const import DOMAIN
+from .models import LD2410BLEData
+
+IS_MOVING_DESCRIPTION = BinarySensorEntityDescription(
+    key="is_motion", device_class=BinarySensorDeviceClass.MOTION
+)
+IS_STATIC_DESCRIPTION = BinarySensorEntityDescription(
+    key="is_static", device_class=BinarySensorDeviceClass.PRESENCE
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the platform for LD2410BLE."""
+    data: LD2410BLEData = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        [
+            IsMovingSensor(data.coordinator, data.device, entry.title),
+            IsStaticSensor(data.coordinator, data.device, entry.title),
```

Comment:
Pass in the description and set `self.entity_description` instead so you only need one class

Reply:
Updated, but I'm not convinced having a single class makes it more readable. It took me quite a while to unpick other examples in order to understand how to do this stuff